frontend/src/utils/Constants.jsx

```diff
@@ -0,0 +1,16 @@
+// MSSC related
+export const SENDER_EDITOR_ROLE = 'mssc:sender_editor';
+export const DEFAULT_SENDER = 'NR.CommonServiceShowcase@gov.bc.ca';
+
+// CMSG related
+export const CMSG_MEDIA_TYPES = ['text/plain', 'text/html'];
```

Comment:
I'm generally not sold on arrays of constants. The idea of constant naming is to be able to know the value of what a constant is/is going to do at the points in the code it's executed.
So, as a hypothetical example, 
`if(x == CMSG_MEDIA_TYPES[1]) {do something}`
doesn't tell me it's going to do something based on that value being html the way that 
`if(x == CMSG_MEDIA_TYPES.HTML)` 
does.
In my opinion at least! Maybe I'm missing something obvious. Don't think it requires a change, just a note.

Reply:
i'll do it in the next PR.